packages/packages/locations/src/hooks/use-fills.ts

```diff
@@ -0,0 +1,10 @@
+import { useMemo } from 'react';
+import { getFills } from '../locations';
+import { Fill } from '../types';
+
+export default function useFills( location : string ): Fill[] {
```

Comment:
```suggestion
export default function useFills( location : Fill['location'] ): Fill[] {
```

Maybe

Reply:
Will be bad for third party plugins types

Reply:
Done